content/fr/0.docs/2.concepts/1.views.md

```diff
@@ -92,11 +63,11 @@ Nous pouvons définir un layout par défaut en ajoutant un fichier `default.vue`
 </template>
 ```
 
-<base-alert type="next">
+::alert{type="next"}
 
 Se référer au [composant Nuxt](/docs/2.x/features/nuxt-components) dans le chapitre des composants.
```

Comment:
To remove 2.x and add /fr on link

Reply:
I think Docus i18n should handle the /fr part actually, as we can host translation under subdomains as well